_data/harnesses/change-topic-partitions-replicas/ksql.yml

```diff
@@ -0,0 +1,139 @@
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/init.adoc
+
+        - change_directory: change-topic-partitions-replicas
+          action: execute
+          file: tutorial-steps/dev/make-dirs.sh
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/make-dirs.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Create the original topic
+      content:
+        - action: execute
+          file: tutorial-steps/dev/create-topic.sh
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/create-topic.adoc
+
+    - title: Describe the original topic
+      content:
+        - action: execute
+          file: tutorial-steps/dev/describe-original-topic.sh
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/describe-original-topic.adoc
+
+    - title: Write the program interactively using the CLI
+      content:
+        - action: docker_ksql_cli_session
+          container: ksqldb-cli
+          docker_bootup_file: tutorial-steps/dev/start-cli.sh
+          column_width: 24
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/start-cli.adoc
+          stdin:
+            - file: tutorial-steps/dev/create-original-stream.sql
+              render:
+                file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/create-inputs.adoc
+
+            - file: tutorial-steps/dev/create-new-stream.sql
+              render:
+                file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/create-new-stream.adoc
+          stdout:
+            directory: tutorial-steps/dev/outputs
+
+    - title: Describe the new topic
+      content:
+        - action: execute
+          file: tutorial-steps/dev/describe-new-topic.sh
+          stdout: tutorial-steps/dev/outputs/actual-describe-from-topic2.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/describe-new-topic.adoc
+
+    - title: Write your statements to a file
+      content:
+        - action: make_file
+          file: src/statements.sql
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/make-src-file.adoc
+
+test:
+  steps:
+    - title: Create the test data
```

Comment:
FYC: Swap the order of creating the test data and starting the producer.  It will be simpler for users to simply copy/paste from the tutorial.

Reply:
Ack, swapping now.